--- config.gcc  2023-09-27 22:42:33.680136900 +0000
+++ config.gcc.patched  2023-09-27 22:41:31.560136900 +0000
@@ -1865,6 +1865,7 @@
        tm_file="${tm_file} i386/unix.h i386/att.h elfos.h newlib-stdint.h i386/i386elf.h"
        ;;
 x86_64-*-elf*)
+       tmake_file="${tmake_file} i386/t-x86_64-elf" # include the new multilib configuration
        tm_file="${tm_file} i386/unix.h i386/att.h elfos.h newlib-stdint.h i386/i386elf.h i386/x86-64.h"
        ;;
 x86_64-*-rtems*)